pkgs/tools/misc/nix-direnv/default.nix

```diff
@@ -2,14 +2,8 @@
 , stdenv
 , fetchFromGitHub
 , gnugrep
-, nixStable
-, nixUnstable
-, enableFlakes ? false
```

Comment:
This flag should be kept until the home-manager module is removed for backwards compatibility: https://github.com/nix-community/home-manager/commit/e0f2949c981a636ff1a75bd075e2a614af3bafb1#diff-eb8eb2f4b79093c79e0d41d3d83055d80638db98059567e66005a144c8c9df8dR79

Reply:
Making a PR there.

Reply:
https://github.com/nix-community/home-manager/pull/2458

Reply:
Still should be kept for a short time in case nixpkgs and home-manager is not in sync. It can be a no-op so we don't loose anything.

Reply:
Alright, left in as a noop